src/lib/ScannerCommand.ts

```diff
@@ -1,27 +1,23 @@
 import {SfdxCommand} from '@salesforce/command';
 import {FilterType, RuleFilter} from './RuleFilter';
 import {uxEvents} from './ScannerEvents';
-import {initContainer} from '../ioc.config';
-
-// Bootstrap the ioc container
-initContainer();
 
 export abstract class ScannerCommand extends SfdxCommand {
 
 	protected buildRuleFilters(): RuleFilter[] {
 		const filters: RuleFilter[] = [];
 		// Create a filter for any provided categories.
-		if ((this.flags.category || []).length > 0) {
+		if (this.flags.category && this.flags.category.length) {
```

Comment:
Switched to this format and removed the comment. Let me know thoughts on the change.

Reply:
Looks fine to me.